Check [WILDCARD]/test/fail.ts
running 10 tests from ./test/fail.ts
test 0 ... FAILED ([WILDCARD])
test 1 ... FAILED ([WILDCARD])
test 2 ... FAILED ([WILDCARD])
test 3 ... FAILED ([WILDCARD])
test 4 ... FAILED ([WILDCARD])
test 5 ... FAILED ([WILDCARD])
test 6 ... FAILED ([WILDCARD])
test 7 ... FAILED ([WILDCARD])
test 8 ... FAILED ([WILDCARD])
test 9 ... FAILED ([WILDCARD])

 ERRORS 

test 0 => ./test/fail.ts:1:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:2:9

test 1 => ./test/fail.ts:4:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:5:9

test 2 => ./test/fail.ts:7:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:8:9

test 3 => ./test/fail.ts:10:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:11:9

test 4 => ./test/fail.ts:13:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:14:9

test 5 => ./test/fail.ts:16:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:17:9

test 6 => ./test/fail.ts:19:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:20:9

test 7 => ./test/fail.ts:22:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:23:9

test 8 => ./test/fail.ts:25:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:26:9

test 9 => ./test/fail.ts:28:6
error: Error
  throw new Error();
        ^
    at [WILDCARD]/test/fail.ts:29:9

 FAILURES 

test 0 => ./test/fail.ts:1:6
test 1 => ./test/fail.ts:4:6
test 2 => ./test/fail.ts:7:6
test 3 => ./test/fail.ts:10:6
test 4 => ./test/fail.ts:13:6
test 5 => ./test/fail.ts:16:6
test 6 => ./test/fail.ts:19:6
test 7 => ./test/fail.ts:22:6
test 8 => ./test/fail.ts:25:6
test 9 => ./test/fail.ts:28:6

FAILED | 0 passed | 10 failed ([WILDCARD])

error: Test failed
